package.json

```diff
@@ -54,6 +54,9 @@
     "test": "NODE_ENV=test jest"
   },
   "jest": {
+    "globals": {
+      "__DEV__": true
```

Comment:
`global === window`? I'm happy to do that and would prefer the explicitness. The initial reason I moved it here was that my babel dev transform was trying to transform this and the quickest fix was to just move the definition.